spikeextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -587,7 +587,7 @@ def add_electrodes(recording: se.RecordingExtractor, nwbfile=None, metadata: dic
                 )
 
     @staticmethod
-    def add_electrical_series(recording: se.RecordingExtractor, nwbfile=None, metadata: dict = None):
+    def add_electrical_series(recording: se.RecordingExtractor, nwbfile=None, metadata: dict = None, buffer_mb=500):
```

Comment:
I have added the optional buffer_mb as an extra argument here; but reconciling this with the general write_recording(), as well as the nwb-conversion-tools pipeline will take some thought.

Reply:
Can you call it `chunk_mb` so we keep the same syntax?

Reply:
It's called `buffer` instead of `chunk` in the `DataChunkIterator` that this function actually uses, so I'd prefer to use the convention that the downstream functions use. This function doesn't directly use the `write_to_binary_dat_format()` which is what uses the `chunk` naming convention.